README.md

```diff
@@ -129,7 +168,7 @@ SQL database. If you need an example, you can use `people` and `wishes` models.
 Your answer...
 
 ```
-Replace this with your answer
+a schema is the structure for a table in our database.  a one to many relationship would be
```

Comment:
👍 on what a schema is. Looking for a one to many relationship is described with a foreign key on records in the many table referring to records in the one table